securedrop/crypto_util.py

```diff
@@ -47,7 +47,7 @@ def do_runtime_tests():
 
 gpg = gnupg.GPG(binary='gpg2', homedir=config.GPG_KEY_DIR)
 
-words = open(config.WORD_LIST).read().rstrip('\n').split('\n')
+language2words = {}
```

Comment:
Hmm, what is `language2words` intended for?

Reply:
Added a comment above to clarify:

    # map code for a given language to a localized wordlist